### DISCUSSION TITLE
Updates & Deletes from Netlify Not Happening

### DISCUSSION THREAD

**Discussion by Mashiane** [pockethost/pockethost#140] [Q&A]
Hi

I just noted that I cant udpate / delete anymore.

![updatesdeletes](https://user-images.githubusercontent.com/5799306/209452098-77844e2d-8da8-4316-a960-2e9db4b706bb.png)

Any idea of a work around?

I am using the JavaScript SDK in my app.

Thank you so much.

**Comment by benallfree** [comment]
@Mashiane That's probably due to my sloppy v0.5.6 update. I believe I have corrected the issue, please try again and let me know if problems persist.
